internal/common/tsconfig.bzl

```diff
@@ -161,7 +161,7 @@ def create_tsconfig(ctx, files, srcs,
   # Keep these options in sync with those in playground/playground.ts.
   compiler_options = {
       # De-sugar to this language level
-      "target": "es5" if devmode_manifest or ctx.attr.runtime == "nodejs" else "es6",
+      "target": "es2015" if devmode_manifest or ctx.attr.runtime == "nodejs" else "es6",
```

Comment:
`es2015` is just an alias for `es6`

https://github.com/Microsoft/TypeScript/blob/7a322d72f76f10e4433905853f6fe1370d98ea83/src/parser/commandLineParser.ts#L150

you say there was some behavior difference when you restored the trinary? I don't see how that's possible...

Reply:
I just tried 3 tests doing `bazel test ...`: `"target": "es6",`, `"target": "es2015",`, `"target": "es2015" if devmode_manifest or ctx.attr.runtime == "nodejs" else "es6",`
es6 fails with
>Expected library.js to contain 'export const cool = 1;' but was

es2015 fails with
>external/build_bazel_rules_typescript/internal/tsetse/language_service_plugin.ts(37,1): error TS1203: Export assignment cannot be used when targeting ECMAScript modules. Consider
using 'export default' or another module format instead.

So only the turnery is passing all tests.
My guess is that it looks like downstream rules are basing their `"module"` off a string comparison of the `target`, comp[lete guess but obth of those errors looks like `module` issues (edited)